### PR TITLE
Data contribution section

### PR DESCRIPTION
**Pull request by teolemon**
I just wrote a data contribution section for the Dart package, and I felt it would be useful here.

### CODE REVIEW

File: lang/en/texts/data.html [hunk 1]
```diff
@@ -110,6 +110,28 @@ <h2>Android and iPhone mobile app</h2>
...
 <p>They will let you consume data, as well as let your users contribute new data</p>
+
+<h3>General principles</h3>
+<ul>
+<li>You can look for information about products, including many useful computed values.</li>
+<li>If you can't get the information on a specific product, you can get your user to send photos and data, that will then be processed by Open Food Facts AI and contributors to get the computed result you want to show them.</li>
+<li>You can also implement the complete flow so that they get immediately the result with some effort on their side.</li>
+</ul>
+
+<h3>If your users do expect a result immediately (eg. Inventory apps)</h3>
```
Comment (stephanegigandet) [inline lang/en/texts/data.html:121]:
```suggestion
<h3>If your users do expect a result immediately (e.g. Inventory apps)</h3>
```

File: lang/en/texts/data.html [hunk 2]
```diff
@@ -110,6 +110,28 @@ <h2>Android and iPhone mobile app</h2>
...
+<ul>
+<li>Send photos (front/nutrition/ingredients): most painless thing for your users</li>
+<li>The Open Food Facts AI Robotoff will generate some derived data from the photos</li>
+<li>Overtime, other apps, and the Open Food Facts community will fill the data gaps</li>
+</ul>
+
+<h3>If your users expect a result immediately (eg. Nutrition apps)</h3>
```
Comment (stephanegigandet) [inline lang/en/texts/data.html:128]:
```suggestion
<h3>If your users expect a result immediately (e.g. Nutrition apps)</h3>
```

**Review by stephanegigandet — APPROVED** [review]

File: lang/en/texts/data.html [hunk 3]
```diff
@@ -110,6 +110,28 @@ <h2>Android and iPhone mobile app</h2>
...
+<li>You can look for information about products, including many useful computed values.</li>
+<li>If you can't get the information on a specific product, you can get your user to send photos and data, that will then be processed by Open Food Facts AI and contributors to get the computed result you want to show them.</li>
+<li>You can also implement the complete flow so that they get immediately the result with some effort on their side.</li>
+</ul>
+
+<h3>If your users do expect a result immediately (e.g. Inventory apps)</h3>
+<ul>
+<li>Send photos (front/nutrition/ingredients): most painless thing for your users</li>
+<li>The Open Food Facts AI Robotoff will generate some derived data from the photos</li>
+<li>Overtime, other apps, and the Open Food Facts community will fill the data gaps</li>
```
Comment (hangy) [inline lang/en/texts/data.html:125]:
```suggestion
<li>Over time, other apps, and the Open Food Facts community will fill the data gaps</li>
```

File: lang/en/texts/data.html [hunk 4]
```diff
@@ -110,6 +110,28 @@ <h2>Android and iPhone mobile app</h2>
...
+</ul>
+
+<h3>If your users expect a result immediately (e.g. Nutrition apps)</h3>
+<ul>
+<li>Send nutrition facts + category > get Nutri-Score</li>
+<li>Send ingredients > get the NOVA group (about food ultra-processing), additives, allergens, normalized ingredients, vegan, vegetarian…</li>
+<li>Send category + labels > get soon the Eco-Score (about environmental impact)</li>
```
Comment (hangy) [inline lang/en/texts/data.html:132]:
```suggestion
<li>Send nutrition facts + category &gt; get Nutri-Score</li>
<li>Send ingredients &gt; get the NOVA group (about food ultra-processing), additives, allergens, normalized ingredients, vegan, vegetarian&hellip;</li>
<li>Send category + labels &gt; soon get the Eco-Score (about environmental impact)</li>
```

File: lang/en/texts/data.html [hunk 5]
```diff
@@ -110,6 +110,28 @@ <h2>Android and iPhone mobile app</h2>
...
 <p>They will let you consume data, as well as let your users contribute new data</p>
+
+<h3>General principles</h3>
+<ul>
+<li>You can look for information about products, including many useful computed values.</li>
+<li>If you can't get the information on a specific product, you can get your user to send photos and data, that will then be processed by Open Food Facts AI and contributors to get the computed result you want to show them.</li>
+<li>You can also implement the complete flow so that they get immediately the result with some effort on their side.</li>
+</ul>
+
+<h3>If your users do expect a result immediately (e.g. Inventory apps)</h3>
```
Comment (stephanegigandet) [inline lang/en/texts/data.html:121]:
```suggestion
<h3>If your users do not expect a result immediately (e.g. Inventory apps)</h3>
``` 
?

**Pull request merged**